src/NumFieldOrd/NfOrd/Elem.jl

```diff
@@ -193,7 +193,7 @@ The minimal polynomial of $a$. The parent of the polynomial can be supplied
 as a first argument.
 """
 function minpoly(R::ZZPolyRing, a::AbsNumFieldOrderElem)
-  return R(minpoly(Globals.Qx, elem_in_nf(a)))
+  return minpoly(R, elem_in_nf(a))
```

Comment:
Is this working right, though? Before it invoked `minpoly` recursively with a `QQPolyRing`, now with a `ZZPolyRing` -- is the latter really supported?

Perhaps a bunch of methods need to be extended to allow for `Union{QQPolyRing,ZZPolyRing}`

Reply:
> is the latter really supported?

Yes, it should be in line 535.